src/macros.jl

```diff
@@ -503,6 +503,28 @@ and
 @transform(df, a = :x, b = :y)
 ```
 
+`@transform` uses the syntax `@byrow` to wrap transformations in
+the `ByRow` function wrapper from DataFrames, enabling broadcasting
+and more. For example, the call
```

Comment:
"and more" isn't super explicit. Maybe keep exporting `@byrow` so that you can attach a docstring to it with more details (copying what appears in the manual)? Then I wouldn't mention `ByRow` here to keep things simple.